code/__DEFINES/shuttles.dm

```diff
@@ -13,6 +13,7 @@
 #define EMERGENCY_IDLE_OR_RECALLED (SSshuttle.emergency && ((SSshuttle.emergency.mode == SHUTTLE_IDLE) || (SSshuttle.emergency.mode == SHUTTLE_RECALL)))
 #define EMERGENCY_ESCAPED_OR_ENDGAMED (SSshuttle.emergency && ((SSshuttle.emergency.mode == SHUTTLE_ESCAPE) || (SSshuttle.emergency.mode == SHUTTLE_ENDGAME)))
 #define EMERGENCY_AT_LEAST_DOCKED (SSshuttle.emergency && SSshuttle.emergency.mode != SHUTTLE_IDLE && SSshuttle.emergency.mode != SHUTTLE_RECALL && SSshuttle.emergency.mode != SHUTTLE_CALL)
+#define EMERGENCY_ALMOST_HERE (SSshuttle.emergency && (SSshuttle.emergency.mode == SHUTTLE_CALL) && (SSshuttle.emergency.timeLeft(1) <= 2 MINUTES))
```

Comment:
Would it still pop up after shuttle left?

Reply:
There is another `return` for `EMERGENCY_ESCAPED_OR_ENDGAMED` check the code in dynamic.

Reply:
Honestly, we should prevent dynamic from spawning rullsets if the shuttle is coming at all

Reply:
I have upped it to 10 minutes, the reason I would be against if the shuttle is coming at all is for the recalls.